### PR TITLE
Nominate Andres Gomez Coronel for 2021H1 SSC Election

### PR DESCRIPTION
**Pull request by bleggett**

### CODE REVIEW

**Comment by bleggett** [comment]
Tagging @Andres-GC for consent purposes.

File: ssc/elections/2021H1/ANDRES_GOMEZ_CORONEL.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+I'd like to nominate Andres to the SSC - Andres has done quite a bit of work adding to and improving code examples 
+and docs, as well as working with contributors to get doc and example PRs merged.
+
+I think Andres would make a strong SSC nominee because in addition to an understanding of the core technicals, 
+Andres is familiar with how people consume, use and understand SPIFFE, and can bring that context to the SSC.
+
+**GitHub Handle:** @Andres-GC   
+**Email Address:** NOMINEE\_EMAIL\_ADDRESS\_HERE  
+**LinkedIn Profile:** LINK\_TO\_NOMINEE\_LINKEDIN\_HERE  
+**Current Affiliation:** N/A
```
Comment (ajessup) [inline ssc/elections/2021H1/ANDRES_GOMEZ_CORONEL.md:12]: Current affiliation is Hewlett Packard Enterprise

File: ssc/elections/2021H1/ANDRES_GOMEZ_CORONEL.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+I'd like to nominate Andres to the SSC - Andres has done quite a bit of work adding to and improving code examples 
+and docs, as well as working with contributors to get doc and example PRs merged.
+
+I think Andres would make a strong SSC nominee because in addition to an understanding of the core technicals, 
+Andres is familiar with how people consume, use and understand SPIFFE, and can bring that context to the SSC.
+
+**GitHub Handle:** @Andres-GC   
+**Email Address:** NOMINEE\_EMAIL\_ADDRESS\_HERE  
+**LinkedIn Profile:** LINK\_TO\_NOMINEE\_LINKEDIN\_HERE
```
Comment (ajessup) [inline ssc/elections/2021H1/ANDRES_GOMEZ_CORONEL.md:11]: Public LinkedIn profile is https://www.linkedin.com/in/andresgomezcoronel/

**Review by ajessup — COMMENTED** [review]
Adds some missing affiliation information

**Comment by Andres-GC** [comment]
> Tagging @Andres-GC for consent purposes.

Hi Ben.

Thank you very much for thinking in me and taking the time to create the PR nominating me.
After thinking a lot I've decided to reject the nomination. I'm sorry I won't honor your nomination but I think that are other people with better conditions to take the place this time.

Again, thank you for nominate me.

Regards.
Andrés.

**Pull request closed without merging**